index.bs

```diff
@@ -691,21 +691,124 @@ An {{MLOperand}} represents an intermediary graph being constructed as a result
+
+#### The {{MLOperand}} constructor #### {#api-mloperand-constructor}
+The [=new=] {{MLOperand}} constructor steps are:
+1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, throw a "{{SecurityError}}" {{DOMException}} and abort these steps.
+1. Let |builder| be the first argument.
+1. If |builder| is not an instance of {{MLGraphBuilder}}, throw a "{{TypeError}}" and abort these steps.
+1. Set [=this=]'s {{MLOperand/[[builder]]}} to |builder|.
+1. Let |kind| be the second argument.
+1. If |kind| is not an {{MLOperandKind}}, throw a "{{TypeError}}" {{DOMException}} and abort these steps.
+1. Set [=this=]'s {{MLOperand/[[kind]]}} to |kind|.
+1. Let |type| be the third argument.
+1. If |type| is not an {{MLOperandType}}, throw a "{{TypeError}}" {{DOMException}} and abort these steps.
+1. Set [=this=]'s {{MLOperand/[[type]]}} to |type|.
+1. If |kind| is `"constant"`:
+    1. Let |name| be the fourth argument.
+    1. If |name| is not a [=string=], throw a "{{TypeError}}" {{DOMException}} and abort these steps.
+    1. Set [=this=]'s {{MLOperand/[[name]]}} to |name|.
+    1. Let |desc| be the fifth argument.
+    1. If |desc| is not an {{MLOperandDescriptor}}, throw a "{{TypeError}}" {{DOMException}} and abort these steps.
+    1. If |type| is not equal to |desc|'s {{MLOperandDescriptor/type}}, throw a "{{DataError}}" {{DOMException}} and abort these steps.
+    1. Set [=this=]'s {{MLOperand/[[dimensions]]}} to |desc|.{{MLOperandDescriptor/dimensions}}.
+    1. Let |bufferView| be the sixth argument.
+    1. If |bufferView| is not an {{MLBufferView}}, throw a "{{DataError}}" {{DOMException}} and abort these steps.
+    1. If |type| is not equal to |bufferView|'s [=element type=], throw a "{{DataError}}" {{DOMException}} and abort these steps.
+    1. If the [=byte length=] of |desc| is not supported by the implementation, throw a "{{DataError}}" {{DOMException}} and abort these steps.
+    1. If the [=byte length=] of |desc| is not equal with |bufferView|.\[[ByteLength]], throw a "{{DataError}}" {{DOMException}} and abort these steps.
+    1. Set [=this=]'s {{MLOperand/[[tensorData]]}} to |bufferView|.
+1. Otherwise, of |kind| is `"input"`:
+    1. Let |name| be the fourth argument.
+    1. If |name| is not a [=string=], throw a "{{TypeError}}" {{DOMException}} and abort these steps.
+    1. Set [=this=]'s {{MLOperand/[[name]]}} to |name|.
+    1. Let |desc| be the fifth argument.
+    1. If |desc| is not an {{MLOperandDescriptor}}, throw a "{{TypeError}}" {{DOMException}} and abort these steps.
+    1. If |type| is not equal to |desc|'s {{MLOperandDescriptor/type}}, throw a "{{DataError}}" {{DOMException}} and abort these steps.
+    1. If the [=byte length=] of |desc| is not supported by the implementation, throw a "{{DataError}}" {{DOMException}} and abort these steps.
+    1. Set [=this=]'s {{MLOperand/[[dimensions]]}} to |desc|.{{MLOperandDescriptor/dimensions}}.
```

Comment:
Ditto, may need to create the platform object representing this "input" MLOperand.